app/components/project-long-description.js

```diff
@@ -1,13 +1,49 @@
 import Ember from 'ember';
 
+/**
+  `project-long-description` displays and allows editing of the long
+  long description for the project.
+
+  ## Default usage
+  ```handlebars
+    {{project-long-description project=project}}
+  ```
+
+  @module Components
+  @extends Ember.Component
+  @class project-long-description
+ */
+
 export default Ember.Component.extend({
   classNames: ['project-long-description'],
 
+  /**
+    A service that returns the users credentials.
+
+    @property credentials
+    @type Ember.Service
+   */
   credentials: Ember.inject.service(),
 
+  /**
+    @property inEditMode
+    @type Boolean
+   */
   inEditMode: false,
+
+  /**
+    @property descriptionIsBlank
+    @type Boolean
+   */
   descriptionIsBlank: false,
 
+  /**
+    Gets whether the editor should be displayed or not based on having no
```

Comment:
I personally prefer `Returns` over `Gets`, but that is 💄 so I'll leave it up to you to decide